src/connection.go

```diff
@@ -194,3 +200,13 @@ func (r redisConn) GetRawCustomKeys(databaseKeys map[string][]string) (map[strin
 
 	return customKeysMetric, nil
 }
+
+// getCommand returns alias of 'command' if exists
+// Redis server security model supports disabling/renaming of specific commands
+// Ref: https://redis.io/topics/security
+func (r redisConn) getCommand(command string) string {
```

Comment:
I'd suggest to rename this to just `command`, to avoid the `get*` java-ism :)

Reply:
agreed